device-types/Intel/H2312XXLR2.yaml

```diff
@@ -0,0 +1,32 @@
+---
+manufacturer: Intel
+model: H2312XXLR2
+slug: intel-h2312xxlr2
+u_height: 2
+is_full_depth: true
+comments: |
+  [H2312XXLR2 Specifications on intel.com](https://www.intel.com/content/www/us/en/products/sku/87573/intel-server-chassis-h2312xxlr2/specifications.html)
+subdevice_role: parent
+device-bays:
+  - name: Node 1
+    label: Blade 1
+  - name: Node 2
+    label: Blade 2
+  - name: Node 3
+    label: Blade 3
+  - name: Node 4
+    label: Blade 4
+power-ports:
```

Comment:
It looks like the power and nic bays (dunno what theyre actually called in Intel land) are actually modular, so these should be removed and created under `module-bays`. Then you should add both the nic bays and the PSUs to the module-types folder

According to the datasheet, the PSUs are `FXX2130PCRPS`

Reply:
Oh wait, looking at your module commit it appears that the modules have the nics on them, so do you put the devices in from the back?? Sorry this is weird compared to the VRTX's I am accustomed to. If those "nic bays" are actually the devices themselves, please let me know and mark this as resolved

Reply:
Yes, the compute blades are put in from the back and they have the network interfaces on them. I had to dig a little deeper on the intel support site and found this neat [Chassis Quick Reference](https://www.intel.com/content/dam/support/us/en/documents/motherboards/server/chassis/h2000/sb/jflabel1.pdf). :)